### PR TITLE
Add linea mainnet

### PR DESCRIPTION
**Pull request by Akaryatrh**
Hi! Following the PRs opened for Linea testnet a few weeks and months ago, we now would like to add support for **Linea mainnet**.
Please keep in mind that RPC endpoints, bridge and block explorer will be available in the upcoming weeks (hence the draft status).

Many thanks for support 🙂

### CODE REVIEW

**Review by Khunyurry — APPROVED** [review]

**Review by alokeutpal — APPROVED** [review]

File: _data/chains/eip155-59144.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{
+  "name": "Linea",
+  "title": "Linea Mainnet",
+  "chain": "ETH",
+  "rpc": [
+    "https://rpc.linea.build",
+    "wss://rpc.linea.build",
+    "https://linea-mainnet.infura.io/v3/${INFURA_API_KEY}",
```
[the remark this answers is not in the document]
Comment (Akaryatrh) [reply _data/chains/eip155-59144.json:8]: I'm not sure to understand your question? What issue are you highlighting?

**Comment by Akaryatrh** [comment]
Side question @ligi, would you be ok if `ethereum-lists` was listed as part of Linea's ecosystem on [linea.build](https://linea.build/) website? If so, would you have a logotype to provide?

(Or maybe it's just the one below? )

**Review by ligi — CHANGES_REQUESTED** [review]
https://lineascan.build is not reachable

**Comment by ligi** [comment]
sure you can list it - and yeah this is the only logo we have

**Review by alokeutpal — APPROVED** [review]

**Comment by Akaryatrh** [comment]
> https://lineascan.build is not reachable
Yep, as said in the PR description, for now neither the RPC endpoints, the bridge or even the block explorer are available. But it's coming soon 🙂

**Comment by Akaryatrh** [comment]
**Update:** changed information for explorers for both testnet and mainnet

**Comment by Akaryatrh** [comment]
FYI @ligi explorer https://lineascan.build and bridge https://bridge.linea.build are now available 🙂 
I then mark this PR as ready.

**Review by ligi — APPROVED** [review]

**Comment by Alamznetwork** [comment]
how do I commit please

**Pull request merged**